### ISSUE TITLE
icons get lost over images

### ISSUE THREAD

**Issue by foundscapes** [OpenArchive/openarchive-ios#61]
can we make them teal or....
![IMG_8764](https://user-images.githubusercontent.com/2231711/55042950-2f40cf80-4ff1-11e9-9347-a2b47ce112f6.PNG)

**Comment by foundscapes** [comment]
Ofc they show up ok in the larger version, but they're hard to see on my iphone 6s

**Comment by tladesignz** [comment]
I could darken the background further.

The icon images use the sizes as per specification. @kckthx Maybe there's a blank border around the icons in their images? Can you remove that and send new SVGs?

**Comment by n8fr8** [comment]
It isn't the visual size of the icons, it is the size of the touchable region. If there was blank space in the images, that would be fine, as those would still be touchable/tappable.

**Comment by n8fr8** [comment]
I think in this case, it is that we need a semi-opaque background for the top bar, so that the icons are never lost. I will talk to @kckthx about this.

**Comment by cstiens** [comment]
> It isn't the visual size of the icons, it is the size of the touchable region. If there was blank space in the images, that would be fine, as those would still be touchable/tappable.

That's correct.

> I think in this case, it is that we need a semi-opaque background for the top bar, so that the icons are never lost. 

We've actually used a gradient in the design mockups:

<img width="841" alt="Screenshot 2019-03-29 10 17 46" src="https://user-images.githubusercontent.com/1434597/55242819-021c3900-520c-11e9-84c5-ce04e6cfda1c.png">

The gradient area is 375x102. @tladesignz I've attached an .svg of the gradient. This is meant to be anchored to the top of the view in this info view as well as to the top of the images in the preview view. Is this implementable?

In this info view, I think we could alternatively do a semi-opaque background for the top bar like Nathan mentioned if the gradient is complicated to implement.

[img-gradient.svg.zip](https://github.com/OpenArchive/openarchive-ios/files/3023295/img-gradient.svg.zip)
